### DISCUSSION TITLE
Search on Pivot Gride

### DISCUSSION THREAD

**Discussion by databayes** [TabularEditor/TabularEditor3#1200] [Ideas]
### Is your feature request related to a problem? Please describe

Is there a way to search and filter when using the pivot grid?
![example](https://github.com/TabularEditor/TabularEditor3/assets/87519321/80e54b60-cdb0-4799-8672-3ecc004d8da3)

### Describe the solution you'd like

I'd like to be able to filter within the pivot grid in the same way I can within a table

### Goals

1.
2.

**Comment by otykier** [comment]
Unfortunately we don't have free text search in the Pivot Grid, but it's a good suggestion, so we'll add it to the backlog.

We do have incremental search, though, so if you start typing while the filter box is showing, you can quickly jump to items that start with certain characters:

<img width="251" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/8976200/e15691ee-f454-4d7d-8964-ba902f27d9db">

**Comment by databayes** [comment]
Brilliant thank you.
Am going to try to challenge myself to capture little bits of learning like this as "shorts" so if you don't mind, I'll make a small video to share with community 🙂
There's so many bits you learn everyday

**Reply by mlonsk** [reply]
Of course, go for it :)

**Comment by databayes** [comment]
you don't actually need to press anything to search, it does it automatically 🙂
